app/src/main/java/com/sdp/movemeet/EditProfileActivity.java

```diff
@@ -88,14 +88,9 @@ private void assignViewsAndAdjustData() {
 
     private void loadRegisteredUserProfilePicture(String userId) {
         progressBar.setVisibility(View.VISIBLE);
-        StorageReference profileRef = storageReference.child("users/"+userId+"/profile.jpg");
-        profileRef.getDownloadUrl().addOnSuccessListener(new OnSuccessListener<Uri>() {
-            @Override
-            public void onSuccess(Uri uri) {
```

Comment:
Why did you remove this part? Was it refactored into FirebaseInteraction?

Reply:
That's exactly this! This is because I use the same method in ActivityDescriptionActivity and and EditProfileActivity to pick a picture from the Library.